wled00/wled.h

```diff
@@ -506,6 +506,13 @@ WLED_GLOBAL WS2812FX strip _INIT(WS2812FX());
 // Usermod manager
 WLED_GLOBAL UsermodManager usermods _INIT(UsermodManager());
 
+// Status LED
+#if STATUSLED && STATUSLED != LEDPIN
+  WLED_GLOBAL unsigned long ledStatusLastMillis _INIT(0);
+  WLED_GLOBAL int ledStatusType _INIT(0); // current status type - corresponds to number of blinks per second
```

Comment:
As for the type used (currently `int`), I'm sure something else could be used. This is an integer that can currently be 0, 2 or 4, so something like an `unsigned short`, should be possible, but I could not get it to work as I kept getting a redefinition error. Any pointers here would be great.